news/2020-08-03-mappers-report-july.md

```diff
@@ -0,0 +1,183 @@
+---
+layout: post
+title: "The Mappers' Report: July"
+date: 2020-08-07 12:00:00 +0000
+---
+
+Good news, everyone! We think we've perfected a news post that will inform you about everything mapping and modding related going on in osu! 
+
+![](/wiki/shared/news/banners/the-mappers-report.jpg)
+
+This month's issue of the Mappers' Report contains the following topics:
+
+- **[Ranking Criteria Changes](#rc-changes)**
+- **[Within the Beatmap Nominators and NAT](#within-bn-nat)**
+- **[Mappers' Showcase](#mappers-showcase)**
+- **[Mapping Contests](#mapping-contests)**
+- **[Conclusion](#conclusion)**
+
+## <a id="rc-changes"></a>Ranking Criteria Changes
+
+For this month, whilst there weren't many major proposals or Ranking Criteria changes, there was a focus on minor wording changes throughout.
+
+### osu!taiko
+ 
+- **[Change the wording for "patterns faster than"](https://osu.ppy.sh/community/forums/topics/1097738)**: This change aims to clear up some ambiguity in the wording of a futsuu difficulty-specific rule.
+
+### osu!mania
+
+- **[Redefining the "stream" terminology](https://osu.ppy.sh/community/forums/topics/1079647])**: Another change mainly looking to make the terminology in the osu!mania Ranking Criteria as clear as possible. This change defines the length at which a pattern is considered a stream.
+- **[Change chord guidelines in normal difficulties](https://osu.ppy.sh/community/forums/topics/1067554)**: This is the only non-wording based change this month. This change, with regards to 4K and 7K normals difficulties, allows mappers to use chords during a long note hold as long as it's being played with the other hand.
+
+## Open Ranking Criteria Proposals
+
+As every month we keep you up to date with the changes to the Ranking Criteria. Additionally we list open proposals so you can give your feedback to them! Below a list of the new proposals since the last issue of the Mappers' Report.
```

Comment:
```suggestion
As every month we keep you up to date with the changes to the Ranking Criteria. Additionally, we list open proposals so you can give your feedback to them! Below a list of the new proposals since the last issue of the Mappers' Report.
```